src/app/widget/sprint-widget/sprint-widget.component.scss

```diff
@@ -0,0 +1,23 @@
+.weather-widget {
```

Comment:
you can remove that class and set class:
:host {
  display: block;
  text-align: center;
  position: relative;
  min-width: 190px;
}

on widget component